homeassistant/components/climacell/sensor.py

```diff
@@ -0,0 +1,153 @@
+"""Sensor component that handles additional ClimaCell data for your location."""
+from __future__ import annotations
+
+import logging
+from typing import Any, Callable, Mapping
+
+from pyclimacell.const import CURRENT
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_NAME,
+    CONF_API_VERSION,
+    CONF_NAME,
+    CONF_UNIT_OF_MEASUREMENT,
+    CONF_UNIT_SYSTEM_IMPERIAL,
+    CONF_UNIT_SYSTEM_METRIC,
+)
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.util import slugify
+
+from . import ClimaCellDataUpdateCoordinator, ClimaCellEntity
+from .const import (
+    ATTR_FIELD,
+    ATTR_IS_METRIC_CHECK,
+    ATTR_METRIC_CONVERSION,
+    ATTR_VALUE_MAP,
+    CC_SENSOR_TYPES,
+    CC_V3_SENSOR_TYPES,
+    DOMAIN,
+)
+
+# mypy: allow-untyped-defs, no-check-untyped-defs
```

Comment:
Where are these needed?

Reply:
I think I copied this over from `weather.py` without thought. Will remove